archinstall/lib/global_menu.py

```diff
@@ -119,7 +119,7 @@ def setup_selection_menu_options(self):
 		self._menu_options['parallel downloads'] = \
 			Selector(
 				_('Parallel Downloads'),
-				lambda preset: add_number_of_parrallel_downloads(preset),
+				lambda preset: add_number_of_parallel_downloads(preset),
```

Comment:
Can we pass the method reference instead of a lambda?
```suggestion
				add_number_of_parallel_downloads,
```

Reply:
Will the `preset` be preserved in the way we call the function at a later stage?

Reply:
we do it the same way with most other settings as well, I don't think it makes much difference